GeoscienceAustralia/dea-docs
# Fixed 'mini' table of contents bug

On the product pages, there is a 'mini' table of contents underneath each tab.

![ksnip_20240205-154654](https://github.com/GeoscienceAustralia/dea-docs/assets/144193171/c208b974-2b96-485f-ada5-ba9ab83f2747)

## Problem

Many of the links in these mini tables of contents were not working anymore after a previous feature was pushed.

## Solution

I rewrote the small script that controls the 'mini' table of contents. By doing so, I fixed this bug and also I refactored it to be much simpler.

## How to test

On the product pages, click the 'mini' table of contents links to check that they work.
